.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
     "python.linting.flake8Enabled": true,
     "python.linting.mypyEnabled": true,
     "python.linting.enabled": true,
-    "python.testing.pytestArgs": [],
+    "python.testing.pytestArgs": ["--cov=python3_pip_skeleton --cov-report xml:cov.xml"],
```

Comment:
This is intentionally all in one string. Strangely, vscode throws a pytest discovery error if these args are in separate elements:
```["--cov=python3_pip_skeleton", "--cov-report xml:cov.xml"]```

makes vscode use

```bash 
./.venv/bin/python ~/.vscode/extensions/ms-python.python-2023.8.0/pythonFiles/testing_tools/run_adapter.py discover pytest -- --rootdir . -s --cache-clear --cov=python3_pip_skeleton "--cov-report xml:cov.xml"
```

while 
``` "python.testing.pytestArgs": ["--cov=python3_pip_skeleton --cov-report xml:cov.xml"],```

gives

```bash
./.venv/bin/python ~/.vscode/extensions/ms-python.python-2023.8.0/pythonFiles/testing_tools/run_adapter.py discover pytest -- --rootdir . -s --cache-clear "--cov=python3_pip_skeleton --cov-report xml:cov.xml"
```
which does work.

Reply:
Does this work?
```suggestion
    "python.testing.pytestArgs": ["--cov=python3_pip_skeleton",  "--cov-report", "xml:cov.xml"],
```

Reply:
It does, my mistake, probably should have tried that